### DISCUSSION TITLE
Where can i find the Value log?

### DISCUSSION THREAD

**Discussion by CommanderV12** [jomjol/AI-on-the-edge-device#1277] [Q&A]
Hello everyone,

I'm looking for the output value log ​​​​of the AI ​​(Value, Rate ..).
I've looked everywhere but just can't find it. I would like to read this log into an excel.

**Comment by jomjol** [comment]
Currently you can only find it within the log file /log/message/DATE
In future version, they also will be separated in a dedicated file /log/data/DATE

The current rolling has this already implemented - but be carefull not stable currently!

**Reply by CommanderV12** [reply]
First of all, many thanks for the quick reply, 

unfortunately there is no usefule data in the log (im only reading NNN.3233 at the moment but they should be logged to or ? 

![image](https://user-images.githubusercontent.com/107116044/200116708-3bd6e45a-9429-4448-a876-b76e66557a83.png)
![image](https://user-images.githubusercontent.com/107116044/200116811-92fe6ad4-72cd-4441-a117-7a1f6934623e.png)

am i missing something?

Thanks a lot in advance

**Reply by CommanderV12** [reply]
im using v11.2.0

**Comment by jomjol** [comment]
yes, you need to switch the logging on in the configuration.

**Reply by CommanderV12** [reply]
![image](https://user-images.githubusercontent.com/107116044/200118851-28c58c33-300a-45f7-94db-4828d27bd202.png)
Thanks, now im getting more informations in the log, but still not the values shown above 

![image](https://user-images.githubusercontent.com/107116044/200118903-432a3a8a-b2b0-44fe-bc32-17e37180e7d8.png)

**Comment by jomjol** [comment]
you are right. this is only logged, if the mqtt is used. 
sorry for this, you need to wait for the next major release (few weeks) or directly go to the rolling (expermimental).

**Reply by CommanderV12** [reply]
ahh that explains it, allright thank you again for the support and great work :)
